rag-and-text2sql-maharishi/query_tools.py

```
@@ -0,0 +1,137 @@
+        {"city_name": "Los Angeles", "population": 3822000, "state": "California"},
+        {"city_name": "Chicago", "population": 2665000, "state": "Illinois"},
+        {"city_name": "Houston", "population": 2303000, "state": "Texas"},
+        {"city_name": "Miami", "population": 449514, "state": "Florida"},
+        {"city_name": "Seattle", "population": 749256, "state": "Washington"},
+    ]
+    try:
+        for row in rows:
+            stmt = insert(city_stats_table).values(**row)
+            with engine.begin() as connection:
+                cursor = connection.execute(stmt)
+    except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)